zproject/configured_settings.py

```diff
@@ -37,7 +37,7 @@
             # TODO: Figure out how to make this not be printed several
             # times, and maybe print the actual keys that are
             # overridden.
-            print("Using custom settings from zproject/custom_dev_settings.py.")
+            # print("Using custom settings from zproject/custom_dev_settings.py.")
```

Comment:
```shell
+ umask go-rw
+ PGPASS_PREFIX='*:*:*:zulip:'
+ PGPASS_ESCAPED_PREFIX='*:\*:\*:zulip:'
+ grep -q '*:\*:\*:zulip:' /home/aman/.pgpass
+ sed -i 's/*:\*:\*:zulip:.*$/*:*:*:zulip:Using custom settings from zproject/custom_dev_settings.py.
e2f9cd4cecf2684af31aefa4c0f621d5aed4b19edffae52a1e076bd8ed9443bf/' /home/aman/.pgpass
sed: -e expression #1, char 69: unknown option to `s'

Subcommand of /home/aman/zulip/tools/lib/provision_inner.py failed with exit status 1: tools/setup/postgresql-init-dev-db
Actual error output for the subcommand is just above this.
```

Reply:
Also, this being printed is very annoying for me. I tried using `os.environ` but that also doesn't seem to work to fix the multiple printed statements.

Reply:
Yeah I think we can't procrastinate reworking this.

Reply:
My thought is we just make `runtornado` have on-startup code to print if that file exists, so it's just once and just in the service code.